### PR TITLE
fix travis builds

### LINKED ISSUE #21: Fix the broken travis ci
Create this place holder issue to signal that I'm working on it :)

### PR DESCRIPTION
**Pull request by lucywang000**
Closes #21 

Also drops python 3.3 support and adds python 3.6 support.

### CODE REVIEW

**Comment by lucywang000** [comment]
Could you please take a look @kmike ?

File: setup.py [hunk 1]
```diff
@@ -49,6 +49,8 @@ def get_version():
         'annotation': [
             'ipython[notebook] >= 4.0',
             'ipywidgets',
+            # See https://github.com/JuliaDocs/Documenter.jl/issues/659
+            'Tornado>=4.0.0,<5.0.0',
```
Comment (kmike) [inline setup.py]: hey! I think we shouldn't restrict Tornado version here, a better fix is to fix an issue with our CI. For some reason installing tornado on Travis works for Tornado itself, but fails in our case - I wonder why. See also: https://github.com/tornadoweb/tornado/issues/2331

File: tox.ini [hunk 2]
```diff
@@ -2,9 +2,14 @@
...
 [tox]
-envlist = py27,py33,py34,py35
+envlist = py27,py34,py35,py36
 
 [testenv]
+basepython =
+    py27: python2.7
+    py34: python3.4
+    py35: python3.5
+    py36: python3.6
```
Comment (kmike) [inline tox.ini]: is this needed? I was thinking it is automatic

File: formasaurus/storage.py [hunk 3]
```diff
@@ -245,16 +245,18 @@ def get_tree(self, path, info=None):
         with open(os.path.join(self.folder, path), "rb") as f:
             return load_html(f.read(), info["url"])
 
-    def check(self):
+    def check(self, verbose=True):
```
Comment (kmike) [inline formasaurus/storage.py]: 👍

File: setup.py (hunk 1 above)
Comment (lucywang000) [reply setup.py]: Makes sense. Now it's solved by using python: 2.7.13 in the travis matrix.

File: tox.ini (hunk 2 above)
Comment (lucywang000) [reply tox.ini]: Yeah, I added it when debugging the python versions problems.

**Comment by lucywang000** [comment]
@kmike All your comments are fixed.

**Comment by kmike** [comment]
Awesome, thanks @lucywang000!

**Pull request merged**